apps/api/src/billing/routes/sign-and-broadcast-tx/sign-and-broadcast-tx.router.ts

```diff
@@ -16,7 +16,8 @@ export const SignTxRequestInputSchema = z.object({
             "/akash.cert.v1beta3.MsgCreateCertificate",
             "/akash.market.v1beta4.MsgCreateLease",
             "/akash.deployment.v1beta3.MsgUpdateDeployment",
-            "/akash.deployment.v1beta3.MsgCloseDeployment"
+            "/akash.deployment.v1beta3.MsgCloseDeployment",
+            "/akash.deployment.v1beta3.MsgDepositDeployment"
```

Comment:
We'll re-add it to enable deposit for managed wallets, so might as well keep it?

Reply:
I'm not sure I follow

Reply:
Oh my bad I must've been tired, I read it like you removed `MsgDepositDeployment`. Ignore my comment!